diagnosisApplication/src/main/interface_adapter/diagnosis/DiagnosisState.java

```diff
@@ -8,134 +8,41 @@ public class DiagnosisState {
 
     private int numDiagnoses = 0;
 
-    private  Integer diagnosis1Ranking = null;
-    private  String diagnosis1Name = null;
-    private  Integer diagnosis1Accuracy = null;
-    private  String diagnosis1Icd = null;
-    private  String diagnosis1ProfName = null;
-    private List<String> diagnosis1Specializations = null;
+    private HashMap<String, Object> diagnosis1;
+    private HashMap<String, Object> diagnosis2;
```

Comment:
Reformatting for more efficient storage of data in hash maps rather than having a multitude of instance variables.